pkg/api/topology.go

```diff
@@ -33,3 +40,52 @@ func Topology(c echo.Context) error {
 
 	return c.JSON(http.StatusOK, results)
 }
+
+func PushTopology(c echo.Context) error {
```

Comment:
I think this should go into mission-control directly,

Reply:
@moshloop Should I import canary-checkers db.Persist Functions there then ?